dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+`spec.dockerImageRepository` field appropriately. For example:
+
+====
+----
+apiVersion: "v1"
+kind: "ImageStream"
+metadata:
+  name: "ruby"
+spec:
+  dockerImageRepository: "openshift/ruby-20-centos7"
+----
+====
+
+After OpenShift synchronizes the tag and image metadata, it looks something like
+this:
+
+====
+----
+kind: "ImageStream"
+apiVersion: "v1"
+metadata:
+  name: "ruby"
+  namespace: "default"
+  selfLink: "/osapi/v1/namespaces/default/imagestreams/ruby"
+  uid: "9990ea5f-f35a-11e4-937e-001c422dcd49"
+  resourceVersion: "53"
+  creationTimestamp: "2015-05-05T19:11:57Z"
+  annotations:
+    openshift.io/image.dockerRepositoryCheck: "2015-05-05T19:12:00Z"
+spec:
+  dockerImageRepository: "openshift/ruby-20-centos7"
+status:
+  dockerImageRepository: "openshift/ruby-20-centos7"
+  tags:
+    -
+      tag: "latest"
+      items:
+        -
+          created: "2015-05-05T19:11:58Z"
+          dockerImageReference: "openshift/ruby-20-centos7:latest"
+          image: "94439378e4546d72ef221c47fe2ac30065bcc3a98c25bc51bed77ec00efabb95"
+    -
+      tag: "v0.4"
+      items:
+        -
+          created: "2015-05-05T19:11:59Z"
+          dockerImageReference: "openshift/ruby-20-centos7:v0.4"
+          image: "c7dbf059225847a7bfb4f40bc335ad7e70defc913de1a28aabea3a2072844a3f"
+----
+====
+
+Querying external registries to synchronize tag and image metadata is not
+currently an automated process. To resynchronize manually, run:
+
+----
+$ oc import-image <image_stream_name>
+----
+
+Within a short amount of time, OpenShift will communicate with the external
+registry to get up to date information about the Docker image repository
+associated with the image stream.
+
+[[insecure-registries]]
+=== Importing Images from Insecure Registries
+
+An image stream can be configured to import tag and image metadata from insecure
+image registries, such as those signed with a self-signed certificate or using
+plain HTTP instead of HTTPS.
+
+To configure this, add the `*openshift.io/image.insecureRepository*` annotation
+and set it to *true*. This setting bypasses certificate validation when
+connecting to the registry:
+
+====
+[source,yaml]
+----
+kind: ImageStream
+apiVersion: v1
+metadata:
+  name: ruby
+  annotations:
+    openshift.io/image.insecureRepository: "true" <1>
+  spec:
+    dockerImageRepository: my.repo.com:5000/myimage
+----
+<1> Set the `*openshift.io/image.insecureRepository*` annotation to *true*
+====
+
+ifdef::openshift-enterprise,openshift-origin[]
+[IMPORTANT]
+====
+The above definition only affects importing tag and image metadata. For this
+image to be used in the cluster (e.g., to be able to do a `docker pull`), each
+node must have Docker configured with the `--insecure-registry` flag. See
+link:../install_config/install/prerequisites.html#host-preparation[Host
+Preparation] for information.
+====
+endif::[]
+
+Additionally there is an option to specify single tag use insecure repository.
```

Comment:
Changed it to: "Additionally, you can specify a single tag using an insecure repository. "